Sources/Sentry/include/SentryANRTracker.h

```diff
@@ -26,16 +30,18 @@ NS_ASSUME_NONNULL_BEGIN
 @interface SentryANRTracker : NSObject
 SENTRY_NO_INIT
 
-- (instancetype)initWithDelegate:(id<SentryANRTrackerDelegate>)delegate
-           timeoutIntervalMillis:(NSUInteger)timeoutIntervalMillis
-             currentDateProvider:(id<SentryCurrentDateProvider>)currentDateProvider
-                    crashWrapper:(SentryCrashWrapper *)crashWrapper
-            dispatchQueueWrapper:(SentryDispatchQueueWrapper *)dispatchQueueWrapper
-                   threadWrapper:(SentryThreadWrapper *)threadWrapper;
+- (instancetype)initWithCurrentDateProvider:(id<SentryCurrentDateProvider>)currentDateProvider
+                               crashWrapper:(SentryCrashWrapper *)crashWrapper
+                       dispatchQueueWrapper:(SentryDispatchQueueWrapper *)dispatchQueueWrapper
+                              threadWrapper:(SentryThreadWrapper *)threadWrapper;
+
+@property (nonatomic, assign) NSTimeInterval timeoutInterval;
```

Comment:
`m`: I think this property should not be exposed as writable. You should pass it in when calling the constructor.

Reply:
This tracker is used from the dependency container for multiple integrations. We don't have the options in the DependencyContainer. And I didn't find a proper approach to do it so. Any suggestion is welcome.